crates/mod-http/src/lib.rs

```diff
@@ -345,7 +395,8 @@ impl LuaUserData for HeaderMapWrapper {
 
         methods.add_meta_method(MetaMethod::Pairs, |lua, this, ()| {
             let stateless_iter =
-                lua.create_function(|lua, (this, key): (HeaderMapWrapper, Option<String>)| {
+                lua.create_function(|lua, (ud, key): (mlua::AnyUserData, Option<String>)| {
+                    let this = ud.borrow::<HeaderMapWrapper>()?;
```

Comment:
this block is also different from previously. Is that intentional/required? It doesn't appear to necessary?